tools/metadata_tools/file_utils.py

```diff
@@ -12,8 +12,12 @@ def safe_read_contents(path_to_file):
             with open(path_to_file, "r", encoding='utf-8') as handle:
                 original_contents = handle.read()
         except UnicodeDecodeError:
-            with open(path_to_file, "r", encoding='utf-16') as handle:
-                original_contents = handle.read()
+            try:
+                with open(path_to_file, "r", encoding='utf-16') as handle:
+                    original_contents = handle.read()
+            except:
+                print(path_to_file)
```

Comment:
Should probably add an informative message to this.
```suggestion
                print("Error reading file: "+path_to_file)
```